soc/riscv/openisa_rv32m1/pinctrl_soc.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Henrik Brix Andersen <henrik@brixandersen.dk>
+ * Copyright (c) 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * @file
+ * OpenISA RV32M1 SOC specific helpers for pinctrl driver
+ */
+
+#ifndef ZEPHYR_SOC_RISCV_OPENISA_RV32M1_PINCTRL_SOC_H_
+#define ZEPHYR_SOC_RISCV_OPENISA_RV32M1_PINCTRL_SOC_H_
+
+#include <devicetree.h>
+#include <zephyr/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @cond INTERNAL_HIDDEN */
```

Comment:
nit: missing endcond at the end, even though this can likely be removed as pinctrl_soc.h is not parsed by Doxygen (and will probably never be)

Reply:
@gmarull Thank you for reviewing. Removed.